orchagent/orchdaemon.cpp

```diff
@@ -1053,8 +1053,10 @@ bool OrchDaemon::warmRestoreAndSyncUp()
 
     syncd_apply_view();
 
-    /* Start dynamic state sync up */
-    gPortsOrch->refreshPortStatus();
+    for (Orch *o : m_orchList)
+    {
```

Comment:
@stepanblyschak where does the refreshPortStatus() happen now?

Reply:
@prgeor Moved to PortsOrch::onWarmBootEnd(), seems it was a good place for refreshPortStatus() to happen as well